### PR TITLE
fix(index) : #MAG-427 fix main page and board creation responsivity, and wrong tab name.

### PR DESCRIPTION
**Pull request by FlorentMr**
## Describe your changes
fix main page and board creation responsivity and added a sidebar drawer for lower sizes (<1024px of width)
fix wrong tab name (myApp instead of Magnéto)

## Checklist tests
Test magneto on different size of screen (for the main page and the board creation)

## Issue ticket number and link
[MAG-427](https://jira.support-ent.fr/browse/MAG-427)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/drawer-sidebar/DrawerSideBar.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    windowDimensions: { width },
+  } = useWindowDimensions();
+  return (
+    <Drawer
+      open={drawer}
+      onClose={toggleDrawer}
+      PaperProps={{
+        sx: {
+          height: width > 767 ? "calc(100% - 67px)" : "calc(100% - 28px)",
+          top: width > 767 ? 67 : 43,
```
Comment (niko-amosse) [inline frontend/src/components/drawer-sidebar/DrawerSideBar.tsx:32]: peut être mettre un ptit com pour expliquer ce calcul farfelu

File: frontend/src/components/header/Header.scss [hunk 2]
```diff
@@ -20,3 +20,9 @@
 .button {
   margin-left: auto;
 }
+
+.mdOrLg {
+  @media (min-width: 1024px) {
```
Comment (niko-amosse) [inline frontend/src/components/header/Header.scss:25]: nous on a mis un ptit fichier pour les breakpoints css ;)

File: frontend/src/components/header/Header.tsx [hunk 3]
```diff
@@ -1,16 +1,34 @@
...
   return (
     <AppHeader>
+      <IconButton
+        color="inherit"
+        aria-label="open drawer"
+        onClick={toggleDrawer}
+        edge="start"
+        sx={{
+          mr: 2,
+          display: isAbove1025px ? "none" : "inline-flex",
```
Comment (niko-amosse) [inline frontend/src/components/header/Header.tsx:27]: gérable en css

File: frontend/src/components/header/Header.scss (hunk 2 above)
Comment (FlorentMr) [reply frontend/src/components/header/Header.scss:25]: Je suis intéressé pour le voir, je le trouve pas sur git

File: frontend/src/components/header/Header.scss [hunk 4]
```diff
@@ -20,3 +20,17 @@
 .button {
   margin-left: auto;
 }
+
+.mdOrLg {
```
Comment (alicedraillard) [inline frontend/src/components/header/Header.scss:24]: pourquoi .mdOrLg ? on peut faire .md .lg

Comment (FlorentMr) [reply frontend/src/components/header/Header.scss:24]: C'était une coquille restée là pour rien, mb

**Pull request merged**